frontend/docker/Dockerfile

```
@@ -0,0 +1,17 @@
+
+FROM node:hydrogen-alpine3.21
+
+WORKDIR /app
+
+ENV PATH /app/node_modules/.bin:$PATH
+
+COPY package*.json .
+COPY . .
+
+RUN npm install
+RUN npm run build
```

Hadolint (2.12.0)

[info] Multiple consecutive `RUN` instructions. Consider consolidation.

(DL3059)